### PR TITLE
Update design on CartToast and refactor to vanilla-extract

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
Update design on CartToast and refactor to vanilla-extract

Screenshot 2024-07-03 at 22 53 34

Screenshot 2024-07-03 at 22 43 15

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4445?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4445** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4445?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4444** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4444?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4437** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4437?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4436** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4436?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gustaveen and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by gustaveen** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4445?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4445** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4445?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4444** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4444?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4437** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4437?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4436** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4436?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gustaveen and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by gustaveen** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4445?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4445** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4445?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4444** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4444?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4437** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4437?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4436** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4436?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gustaveen and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by alebedev — APPROVED** [review]
Great!

File: apps/store/src/components/CartNotification/CartToast.css.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import { style, keyframes } from '@vanilla-extract/css'
+import { minWidth, tokens } from 'ui'
+import { MAX_WIDTH } from '../GridLayout/GridLayout.constants'
+import { MENU_BAR_HEIGHT_DESKTOP } from '../HeaderNew/Header.css'
+import { MENU_BAR_HEIGHT_MOBILE } from '../HeaderNew/HeaderMenuMobile/HeaderMenuMobile.css'
+
+export const bundleDiscountWrapper = style({
```
[the remark this answers is not in the document]
Comment (gustaveen) [reply apps/store/src/components/CartNotification/CartToast.css.ts:7]: Right!

**Comment by gustaveen** [comment]
### Merge activity

* **Jul 4, 7:55 AM EDT**: [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4445) rebased this pull request as part of a merge.

**Pull request merged**